cpu/samd21/periph/i2c.c

```diff
@@ -197,7 +197,7 @@ int i2c_init_master(i2c_t dev, i2c_speed_t speed)
     i2c_poweron(dev);
 
     /* Start timeout if bus state is unknown. */
-    while (!(I2CSercom->STATUS.reg & SERCOM_I2CM_STATUS_BUSSTATE(1))) {
+    while (!(I2CSercom->STATUS.reg & SERCOM_I2CM_STATUS_BUSSTATE_Msk)) {
```

Comment:
please update this to check for equality against `BUSSTATE_UNKNOWN`, to improve readability.
```c
while ((I2CSercom->STATUS.reg & SERCOM_I2CM_STATUS_BUSSTATE_Msk) == BUSSTATE_UNKNOWN)
```